### PR TITLE
Create templates for issues and pull requests

### PR DESCRIPTION
**Pull request by castelao**
With these templates, we can organize the submission process and drive which information is required for each type of goal. For instance, to propose a typo fix might require less information than a proposal to add a new attribute or change the name of a dimension.

This was initially proposed to be included on #87, but I realized that it could be independent of the governance. Smaller PRs with focused objectives are easier to analyze and thus easier to evaluate.

### CODE REVIEW

**Comment by castelao** [comment]
It is worth mentioning that this was inspired by the templates for CF-Conventions.

**Comment by castelao** [comment]
I think that the types of issues that we should cover are:
- typo (A clear typo)
- style (Bad format, spacing, font, ... anything for better reading without changing the visible text)
- bug (An error in the format that requires changing)
- enhancement (An improvement that requires some changes to what already exists. Usually an improvement on an existent feature. For instance, a change on a data type, adding an extra dimension to an existent variable)
- feature (A new improvement that can be added on top of the current format without changes, just additions. For instance, a new variable or new attribute)

What do you think about it?

**Comment by vturpin** [comment]
I like it. It gives some insight to users on how they can contribute.

Few cents on this : 
I imagine that each type of issue will have a different governance rule ? 
To reduce complexity, maybe we could regroup the typo/style under the same reviewing and acceptation rules ? 
Then Bug, 
Then enhancement / feature together ?

**Comment by castelao** [comment]
@vturpin , yes, that's a possibility. The reason I split like that was that style might involve personal taste, while a typo is a rule with an easy and clear answer. For enhancement vs feature, the idea was one requires changes to what is already in use, while the other is just an addition.

Let me combine them and see how it looks.

**Review by callumrollo — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -22,6 +22,10 @@ See issue #XXX for discussion of these changes.
 # Release checklist
 
 - [ ] Approved by at least two members of the committee?
+- [ ] There were modifications after the review approvals? If so, please
+      as reviewers to update their review.
```
Comment (vturpin) [inline .github/PULL_REQUEST_TEMPLATE.md:26]: "ask" instead of "as" ?

**Review by vturpin — APPROVED** [review]

**Comment by castelao** [comment]
This is not perfect but is a starting point.

**Pull request merged**